TestDrivenDevelopmentTest/TestDrivenDevelopment.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using TestDrivenDevelopment;
+using Xunit;
+
+namespace TestDrivenDevelopmentTest
+{
+    public class TestDrivenDevelopment
+    {
+        [Fact]
+        public void PrintCSVFormat1()
+        {
+            var value = new List<int> { 4, 5 };
+            var result = TestDrivenRanges.ProcessSamples(value);
+            foreach (var item in result)
+            {
+                Assert.Equal("4-5, 2", item);
+            }
+        }
+
+        [Fact]
+        public void PrintCSVFormat2()
```

Comment:
Use the test function names to express functionality. E.g., this one can be named `PrintCSVFormatOfTwoRanges`
Such naming helps in figuring out if we missed any test possibilities.

Reply:
sorry, this point was missed. corrected in latest commit